server/src/cm_job_task.cpp

```diff
@@ -7133,9 +7116,14 @@ ts_killtran (nvplist *req, nvplist *res, char *_dbmt_error)
   argv[argc++] = UTIL_OPTION_KILLTRAN;
   if (dbpasswd != NULL)
     {
-      argv[argc++] = "--" KILLTRAN_DBA_PASSWORD_L;
-      argv[argc++] = dbpasswd;
+      if (strcmp (type, "i") == 0 || strcmp (type, "u") == 0 || strcmp (type, "h") == 0 || strcmp (type, "p") == 0
+          || strcmp (type, "s") == 0 || strcmp (type, "q") == 0)
```

Comment:
The q option doesn't need password, so "strcmp (type, "q") == 0" will be removed.

Reply:
I got it. Thank you.